swagger/control/swagger.yaml

```diff
@@ -147,6 +139,81 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/Entity"
+      min_period:
+        description: is the minimum push period frequency for a channel, in seconds.
+        type: number
+        format: float
+      max_size:
+        description: indicates the maximum allowed size of the message, in bytes.
+        type: integer
+        format: int64
     required:
+      - descriptor
+      - token
       - sender
       - recipients
+      - min_period
+      - max_size
+
+  Rel:
+    description: defines the link relation information about a page.
+    type: object
+    properties:
+      href:
+        description: contains a URL.
+        type: string
+    required:
+      - href
+
+  OptionalRel:
+    description: defines the link relation information about a page that might not exist.
+    type: object
+    properties:
+      href:
+        description: contains a URL.
+        type: string
+
+  Rels:
```

Comment:
Rels are actually expected in the header. Since you are including them in the response body (and not response header), you can do away with `OptionalRel` and just set `prev` and `next` as not required fields, couldn't you?

Reply:
You think that's the way to go? Otherwise I can set all rels in the header directly in the code and perhaps only mention their existance in the swagger description of the ChannelsPage.